### PR TITLE
Update WIS2_Message_Format_Schema.yaml

### PR DESCRIPTION
**Pull request by golfvert**
Adjust length and size

### CODE REVIEW

File: WIS2_Message_Format_Schema.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ definitions:
           example: Trierer Strasse 70, 53115 Bonn
         length:
           type: integer
+          # For file links optional length provides the number of bytes in the file
```
Comment (tomkralidis) [inline WIS2_Message_Format_Schema.yaml:31]: Can we make this not a comment but a description?

File: WIS2_Message_Format_Schema.yaml [hunk 2]
```diff
@@ -146,16 +147,17 @@ properties:
             enum:
               - utf-8
               - base64
-          length:
+          size:
             type: integer
             description: Number of bytes contained in the file. Together with the ``integrity`` property, it provides additional assurance that file content was accurately received.
             # RGiraud: Can we limit the size of the embedded data here?
+            # Using size to avoir confusion with length in links array
```
Comment (tomkralidis) [inline WIS2_Message_Format_Schema.yaml:154]: Do we need this comment?

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: WIS2_Message_Format_Schema.yaml (hunk 2 above)
Comment (golfvert) [reply WIS2_Message_Format_Schema.yaml:154]: Description is fine.

Comment (golfvert) [reply WIS2_Message_Format_Schema.yaml:154]: No need for the comment.

**Pull request merged**